instaclustr/resource_cluster.go

```diff
@@ -69,12 +69,24 @@ func resourceCluster() *schema.Resource {
 				Optional: true,
 			},
 
+			"public_contact_addresses": {
```

Comment:
Based on your fix for `PublicAddress` and `PrivateAddress`, it appears that the `public_contact_point` and `private_contact_point` properties have never really worked before. Would it make sense to remove that property and only have these new ones? 

Looping in @peter-toms in on this suggestion as well.

Reply:
yeah, well I was not that sure and did not want to break backwards compatibility just incase it was working for someone, somewhere.. some how. :)

Reply:
I easy if you wish to change it.